workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/Readers/NonBlittableReaderBase.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using Improbable.Gdk.Core.MonoBehaviours;
+using Unity.Entities;
+using UnityEngine;
+
+namespace Improbable.Gdk.Core.GameObjectRepresentation
+{
+    public abstract class NonBlittableReaderBase<TSpatialComponentData, TComponentUpdate>
+        : ReaderBase<TSpatialComponentData, TComponentUpdate>
+        where TSpatialComponentData : Component, ISpatialComponentData
+        where TComponentUpdate : ISpatialComponentUpdate
+    {
+        protected NonBlittableReaderBase(Entity entity, EntityManager entityManager, ILogDispatcher logDispatcher) : base(entity, entityManager, logDispatcher)
```

Comment:
I thought we want to get rid of the non-blittable components. Do we need this?

Reply:
If they are gone then this can be removed indeed